module/VuFind/src/VuFind/Db/Row/UserList.php

```diff
@@ -305,4 +308,118 @@ public function delete($user = false, $force = false)
         // Remove the list itself:
         return parent::delete();
     }
+
+    /**
+     * Get identifier (returns null for an uninitialized or non-persisted object).
+     *
+     * @return ?int
+     */
+    public function getId(): ?int
+    {
+        return $this->id ?? null;
+    }
+
+    /**
+     * Title setter
+     *
+     * @param string $title Title
+     *
+     * @return UserListEntityInterface
+     */
+    public function setTitle(string $title): UserListEntityInterface
+    {
+        $this->title = $title;
+        return $this;
+    }
+
+    /**
+     * Get Title.
+     *
+     * @return string
+     */
+    public function getTitle(): string
+    {
+        return $this->title;
+    }
+
+    /**
+     * Description setter
+     *
+     * @param string $description Description
+     *
+     * @return UserListEntityInterface
+     */
+    public function setDescription(string $description): UserListEntityInterface
+    {
+        $this->description = $description;
+        return $this;
+    }
+
+    /**
+     * Get Description.
+     *
+     * @return string
+     */
+    public function getDescription(): string
+    {
+        return $this->description;
+    }
+
+    /**
+     * Created date setter.
+     *
+     * @param DateTime $dateTime Created date
+     *
+     * @return UserListEntityInterface
+     */
+    public function setCreated(DateTime $dateTime): UserListEntityInterface
+    {
+        return $this->created = $dateTime;
```

Comment:
$this->created is a string, so you need to convert back and forth to DateTime objects to make the created getter and setter work correctly. You can model it on this code:

https://github.com/vufind-org/vufind/pull/3646/files#diff-efc114e5246846d1aee94c828770f09dc54022a69c9bff25b27eb569139f1db3R109-R130